pywhat/what.py

```diff
@@ -89,6 +89,13 @@ def create_filter(rarity, include, exclude):
     "-db", "--disable-boundaryless", is_flag=True, help="Disable boundaryless mode."
 )
 @click.option("--json", is_flag=True, help="Return results in json format.")
+@click.option(
+    "-fn",
+    "--search-filenames",

```

Comment:
This name is confusing, maybe `--include-filenames`?

Reply:
True I will rename it. Is short `fn` okay?

Reply:
It might be confusing since the short is different from the long, if you only knew the long I think you'd expect it to be `-if` :)